app/Console/Command/ExcludeMalformedLabel.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Label\Commands\ExcludeLabel as ExcludeLabelCommand;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use Doctrine\DBAL\Connection;
+use PDO;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+final class ExcludeMalformedLabel extends AbstractCommand
+{
+    private const LABEL_REGEX = '/^[a-zA-Z\d_\-]{2,50}$/';
+
+    private const MAX_RESULTS = 1;
```

Comment:
Maybe tweak this value for better performance? There is no need here to make a SQL call for every single label.